server/src/main/java/org/opensearch/rest/action/admin/cluster/RestGetDecommissionAction.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.rest.action.admin.cluster;
+
+import org.opensearch.action.admin.cluster.decommission.awareness.get.GetDecommissionRequest;
+import org.opensearch.client.Requests;
+import org.opensearch.client.node.NodeClient;
+import org.opensearch.rest.BaseRestHandler;
+import org.opensearch.rest.RestRequest;
+import org.opensearch.rest.action.RestToXContentListener;
+
+import java.io.IOException;
+import java.util.List;
+
+import static java.util.Collections.singletonList;
+import static org.opensearch.rest.RestRequest.Method.GET;
+
+/**
+ * Returns decommissioned attribute information
+ *
+ * @opensearch.api
+ */
+public class RestGetDecommissionAction extends BaseRestHandler {
+
+    @Override
+    public List<Route> routes() {
+        return singletonList(new Route(GET, "/_cluster/decommission/awareness/_status"));
```

Comment:
Do we see other path getting added to `_cluster/decommission/` than `awareness`? If yes, will that route be part of `routes` of `RestGetDecommissionAction` or there will be a separate action for the same? I am asking this because there is a disconnect between action name and route. If all the decommission related routes come under the same action, then the action name makes sense.

Reply:
Currently, there's no plan as such for that. But in case tomorrow we have a feature request for node decommission the path will be easily adapted to implement that. For eg. `_cluster/decommission/node/<node_id>` 

> If yes, will that route be part of routes of RestGetDecommissionAction or there will be a separate action for the same

Didn't give this a thought yet. But IMO, for any such feature request in future to get status of anything decommissioned, we would be able to use the same RestAction.
Let me know what do you think?

Reply:
The same action can be reused when we have node decommissioning feature.